src/style.rs

```diff
@@ -180,6 +194,34 @@ impl Style {
         )
     }
 
+    pub(crate) fn tab_plus(&self, ui: &mut Ui) -> Response {
+        let desired_size = vec2(24.0, 24.0);
+
+        let (rect, response) = ui.allocate_at_least(desired_size, Sense::hover());
+        let response = response.on_hover_cursor(CursorIcon::PointingHand);
+        let rect = rect.shrink(7.5);
```

Comment:
One issue I'm seeing is that on low-DPI screens the + lines aren't well aligned, most likely because of their non-pixel-perfect placement. Some of the misalignment is the result of being placed right next to the rightmost tab, because tabs can have irregular widths:

![image](https://user-images.githubusercontent.com/2311185/191944557-c7ef9e71-df16-48c9-8bbc-e24b529e5cf4.png)

One easy way to do fix this could be sticking the + button to the right side of the tab bar like this:

![image](https://user-images.githubusercontent.com/2311185/191946601-39dc8438-f126-42e2-8e6b-18f8173c3749.png)

To achieve this exact effect I replaced lines 200-202 with:

```rs
let mut rect = ui.available_rect_before_wrap();
rect.min.x = rect.max.x - desired_size.x;
rect = rect.shrink(3.0);
let response = ui
    .allocate_rect(rect, Sense::hover())
    .on_hover_cursor(CursorIcon::PointingHand);
let rect = rect.shrink(4.0);
```

Reply:
Regarding this, I'm thinking of using `egui`'s `Align` struct, but theres also an `Align::Center` variant to be taken care of. Should we use this or spin up something else?

Reply:
I think a new enum would be better here